.github/workflows/publish.yaml

```diff
@@ -0,0 +1,67 @@
+# reusable workflow for publishing all charms in this repo
+name: Publish
```

Comment:
use the reusable workflows from identity-team, copy hydra-operator or similar, both for publish and release

Reply:
I copied from an older commit... my bad